src/requests/getForecast.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-console */
+import axios from "axios";
+
+const getForecast = (
+  searchText,
+  setSelectedDate,
+  setForecasts,
+  setLocation,
+  setErrorMessage
+) => {
+  let endpoint = "https://cmd-shift-weather-app.onrender.com/forecast";
+
+  if (searchText) {
+    endpoint += `?city=${searchText}`;
+  }
+
+  axios
+    .get(endpoint)
+    .then((response) => {
+      setErrorMessage("");
```

Comment:
You can destructure from the response object here as well. I tend to call the response `res` as it's more shorthand and very commonly used.